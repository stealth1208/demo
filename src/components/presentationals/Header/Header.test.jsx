import React from 'react';
import renderer from 'react-test-renderer';
import { Header } from '../index';

it('renders correctly', () => {
    const tree = renderer
      .create(<Header><div>Header menu</div></Header>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });