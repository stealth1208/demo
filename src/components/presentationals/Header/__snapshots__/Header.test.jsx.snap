// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="header"
>
  <div
    className="header__link"
  >
    <a
      href="/super-heroes"
    >
      Super Heroes
    </a>
  </div>
  <div
    className="header__copyright"
  >
    Â© 2019 MARVEL
  </div>
</div>
`;
